WNPRC_EHR/src/org/labkey/wnprc_ehr/TriggerScriptHelper.java

```diff
@@ -99,6 +105,80 @@ public void sendPregnancyNotification(final List<String> ids, final List<String>
 //        }
     }
 
+    public void insertRows(List<Map<String, Object>> insertRows, String schema, String table) throws QueryUpdateServiceException, SQLException, BatchValidationException, DuplicateKeyException {
```

Comment:
It's a public method so it can be used by trigger scripts themselves, or by other methods within the TriggerScriptHelper. I'm not sure what to add as a comment since it's a pretty simple helper method.